radio/src/functions.cpp

```diff
@@ -376,9 +376,7 @@ void evalFunctions()
           case FUNC_SET_TIMER:
             timerSet(CFN_TIMER_INDEX(cfn), CFN_PARAM(cfn));
             break;
-#endif
```

Comment:
Perhaps you could forbid DANGEROUS_MODULE_FUNCTIONS on !ARM boards in CMakeLists.txt, it would avoid the repetition